meta-mender-core/classes/mender-setup-bios.inc

```diff
@@ -1,6 +1,7 @@
 # Make sure that GRUB is flashed to first non-partition table sector.
 MENDER_IMAGE_BOOTLOADER_FILE_DEFAULT_mender-grub_mender-bios = "grub-core.img"
-MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios = "1"
+MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios_mender-image-bios = "1"
+MENDER_IMAGE_BOOTLOADER_BOOTSECTOR_OFFSET_DEFAULT_mender-grub_mender-bios_mender-image-gpt = "34"
```

Comment:
Is this number 34 documented anywhere? Might be good to have a comment reference to that.